el_gato/el_gato.py

```diff
@@ -1253,16 +1253,20 @@ def process_reads(contig_dict: dict, read_info_dict: dict, ref: Ref, outdir: str
         # Try to use depth to determine the right one to use
         # if highest depth locus more than 3* the depth of next highest, keep
         if float(cov_sorted[0][1]) > 3*float(cov_sorted[1][1]):
-            logging.info(f"mean coverage of {cov_sorted[0][0]} is greater. Using {cov_sorted[0][0]}.")
+            logging.info(f"Mean depth of {cov_sorted[0][0]} is greater. Using {cov_sorted[0][0]}.")
+            neuA_call = cov_sorted[0][0]
             for gene in cov_sorted[1:]:
                 del ref.REF_POSITIONS[gene[0]]
         elif float(cov_sorted[0][1]) < 3*float(cov_sorted[1][1]):
-            logging.info(f"mean coverage of {cov_sorted[1][0]} is greater. Using {cov_sorted[1][0]}.")
+            logging.info(f"Mean depth of {cov_sorted[1][0]} is greater. Using {cov_sorted[1][0]}.")
```

Comment:
This check for > 3X depth of coverage isn't to decide whether to use the first or second neuA reference. The `cov_sorted` list is sorted from most depth to least depth. This is a QC check to see if the first index has more than 3X the depth of the second index. If so then we are confident this is the right reference. If not (i.e., the elif or else in your commit), then we aren't sure which reference to use to call the allele sequence and leave neuA as -.

Your change in the elif block would use the reference with the second most depth, which is not what we want.